tests/migration_test.go

```diff
@@ -486,11 +486,23 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 					// check VMI, confirm migration state
 					confirmVMIPostMigration(vmi, migrationUID)
 
-					By("Check if Migrated VMI has updated PodIP")
+					By("Check if Migrated VMI has updated IP and IPs fields")
 					Eventually(func() bool {
 						newvmi, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Get(vmi.Name, &metav1.GetOptions{})
 						Expect(err).ToNot(HaveOccurred(), "Should successfully get new VMI")
 						vmiPod := tests.GetRunningPodByVirtualMachineInstance(newvmi, tests.NamespaceTestDefault)
+
+						if len(newvmi.Status.Interfaces[0].IPs) != len(vmiPod.Status.PodIPs) ||
+							len(newvmi.Status.Interfaces[0].IPs) == 0 {
+							return false
+						}
+
+						for i, ip := range vmiPod.Status.PodIPs {
+							if newvmi.Status.Interfaces[0].IPs[i] != ip.IP {
+								return false
+							}
+						}
```

Comment:
As mentioned before; this could be re-used, right (here, for instance, I don't think comparing the length to 0 is redundant) ?

Reply:
done